src/client/components/WeekWithDetail/index.js

```diff
@@ -87,22 +89,40 @@ function WeekWithDetail(props) {
     return getSubmittedAnswer(questionName, weekData);
   });
 
+  // TODO(kalvin): refactor collapsible accordion item from here and Acknowledgement
+  // in RetroCertsConfirmationPage out to common file
+  const EN_DASH = "–";
   return (
-    <Alert variant="secondary" className="d-flex">
-      <div className="flex-fill">
-        <Trans
-          t={t}
-          i18nKey="retrocerts-week-list-item"
-          values={{ ...dates, weekForUser }}
-        />
-        <WeekConfirmationDetails
-          employers={weekHasEmployers ? weekData.employers : undefined}
-          questionAnswers={questionAnswers}
-          questionKeys={questionKeys}
-          weekString={toWeekString(weekIndex)}
-        />
-      </div>
-    </Alert>
+    <React.Fragment>
+      <Alert variant="secondary" className="d-flex">
+        <div className="flex-fill">
+          <button
+            className="toggleAccordion"
+            onClick={() => setShowDetail(!showDetail)}
```

Comment:
According to the proposal, expanding a new section is supposed to close the other sections.

Reply:
Great catch! I'll fix that. I think this is something we'll want to change for the staff admin app since staff won't be on mobile. cc @nguyen-tina

Reply:
Talked to Tina, we decided to keep it as is, with the ability to open multiple sections. The intention with only opening one was to not make the page too long for mobile users. But that makes it extremely difficult for both desktop and mobile users to compare their answers between weeks, or look at multiple weeks at once. We'll implement expanding them all when printing as a enhancement in the next PR.

Reply:
Thank you ! Sorry I'm not bopping around as responsively on github